packages/template-retail-react-app/app/commerce-api/hooks/useBasket.js

```diff
@@ -123,13 +123,7 @@ export default function useBasket(opts = {}) {
                     throw new Error(response)
                 } else {
                     setBasket(response)
-                    const einsteinProduct = {
```

Comment:
This is not related to this PR specifically, but is there a reason we only send the first item out of a list of item added to cart? 🤔 . Should we send all the items that are added to cart to Einstein?

Reply:
I believe this assumes that only 1 item is being added to the cart whenever the Add to Cart button is clicked. This probably will need to change if we introduce other product types (sets or bundles) in the future.

Reply:
Make sense to me.